_episodes/03-github-pages.md

```diff
@@ -1,21 +1,152 @@
+on a special GitHub IO URL composed from your GitHub name and repository name"
 ---
 
-- push to a GitHub repo
-- activate Pages in the repo settings
-- change something in `index.md`
-- go to landing page
-- make a change in `index.md` and see how landing page updates
-- add a second `.md`
-- add minimal YAML header
+# What is GitHub Pages?
+[GitHub Pages](https://docs.github.com/en/github/working-with-github-pages/about-github-pages) is a free website 
+hosting service by GitHub that takes files from your GitHub repository that is configured as a website, 
+optionally runs the files through a build process, and publishes them as a website or a web page. 
+Any changes you do to the files in your website's GitHub repository 
+will be rendered live in the website.
+
+[Anne's bit]
+
+# Website Configuration
+So far we have seen how to configure your repository to host a website and where to find the rendered website. Let's
+now look into the configuration of the website itself.
+
+## Configuring Parameters via `_config.yml`
+We have already mentioned that GitHub has an engine under the hood to process your files and 
+build functional websites out of your content. 
+This engine is called Jekyll and we will talk more about Jekyll in the following episodes. 
+Jekyll's main configuration options are specified in `_config.yml` file placed in your site’s root directory. 
+`_config.yml` file is written 
+in the language called [YAML](https://yaml.org/) - a human-readable data-serialization language that allows us to define scalar 
+parameters (that contain single values) as well as parameters that contain lists and dictionaries (hashes). 
+
+> ## YAML 
+>
+> [YAML](https://yaml.org/) is a human-readable data-serialization language. It is commonly used for configuration files and in 
+> applications where text data is being stored or transmitted and it is programming language agnostic.  
+>
+{: .callout}
+
+Let's create some configuration parameters for our website.
+
+1. From the GitHub interface, create `_config.yml` file in your site’s root directory.
+2. Add parameters `title`, `description` and `email` to it as:
+
+    ~~~
+    title: "Research Project Website"
+    description: "Research Project Description"
+    email: team@my.research.org
+    ~~~  
+    {: .language-yaml}
+
+3. Commit your changes.
+
+Site-wide information and configuration settings from 
+`_config.yml` are made available as `site.PARAMETER_NAME` variable in every page/file within the website/repository. 
+There is a number of 
+[predefined site-wide variables](https://jekyllrb.com/docs/variables#site-variables) available to you 
+(such as `site.title` and `site.email`). Others (such as `site.description`) you can define yourself. 
+
+To access the values of configuration parameters within Markdown files, you have to enclose them in double curly 
+braces: {{ "{{ and  " }}}}. Let's see this in action.
```

Comment:
```suggestion
braces: {% raw %}{{ and  }}{% endraw %}. Let's see this in action.
```

can we use `{% raw %}`/`{% endraw %}` tags in cases like this, instead of `{{ "`/`" }}`? Neither solution is pretty, but writing Jekyll about writing Jekyll was always going to require some ugliness 😆 I find the raw/endraw approach mildly less unreadable.

Reply:
Fantastic, I was not aware of this and struggled to find the solution for adding 'raw' liquid code in text.